src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
```

Comment:
The parameter 'days' could be more descriptive. Consider renaming it to 'numberOfDays' to make its purpose clearer.